app/controllers/api_controller/error_handler.rb

```diff
@@ -35,7 +35,10 @@ def api_error(kind, message, klass, backtrace, status)
       # We don't want to return the stack trace, but only log it in case of an internal error
       api_log_error("\n\n#{backtrace}") if kind == :internal_server_error && !backtrace.empty?
 
-      render :json => {:error => err}, :status => status
+      json = {:error => err}
+      json[:backtrace] = backtrace if Rails.env.test?
+      render :json => json, :status => status
+
```

Comment:
:thumbsup:

Would the backtrace ever be empty?

I think this would be good for 400s too as this would mean the difference between grepping for an error message and just jumping straight to the line.